src/signals/incident/definitions/wizard-step-2-vulaan/wonen.ts

```diff
@@ -778,6 +778,200 @@ const vakantieverhuur = {
   },
 }
 
+const verhuurderschap = {
+  extra_wonen_verhuurderschap_onderwerp: {
+    meta: {
+      ifOneOf: {
+        subcategory: 'goed-verhuurderschap',
+      },
+      label: 'Over welk onderwerp gaat uw melding?',
+      shortLabel: 'Onderwerp melding',
+      pathMerge: 'extra_properties',
+      values: {
+        discriminatie: 'Discriminatie',
+        intimidatie: 'Intimidatie',
+        huurcontract: 'Huurcontract',
+        bemiddelingskosten: 'Bemiddelingskosten',
+        arbeidsmigrant: 'Arbeidsmigrant',
+        servicekosten: 'Servicekosten',
+        overige: 'Overige (iets anders namelijk)',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.RadioInput,
+  },
+
+  // Discriminatie
+  extra_wonen_verhuurderschap_discriminatie: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label:
+        'Bent u gediscrimineerd tijdens het verhuurproces? We bedoelen in dit geval of u een woning niet heeft gekregen omdat u tot een specifieke groep behoort. Op welk grond bent u gediscrimineerd? Kies uit:',
+      shortLabel: 'Discriminatie groep',
+      pathMerge: 'extra_properties',
+      values: {
+        ras: 'Ras',
+        geloofsovertuiging: 'Geloofsovertuiging',
+        politieke_overtuiging: 'Politieke overtuiging',
+        gesalcht: 'Geslacht',
+        nationaliteit: 'Nationaliteit',
+        seksuele_gerichtheid: 'Seksuele gerichtheid',
+        burgelijke_staat: 'Burgerlijke staat',
+        handicap: 'Handicap',
+        chonische_ziekte: 'Chronische ziekte',
+        anders: 'Een andere reden, namelijk ',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.CheckboxInput,
```

Comment:
Zou dit niet ook een radioinput moeten zijn? Nu denk ik dat ik ook niet een checkbox hoef aan te vinken terwijl het wel required is.

Reply:
Don't think so, since you can be discriminated on multiple fields. However, they did not say what should be radio, input field or checkbox. So you can leave this out of the review.